plugins/zeneyer-auth/includes/API/class-rest-routes.php

```
@@ -247,8 +247,9 @@ public static function register($request)
         $email = $request->get_param('email');
         $password = $request->get_param('password');
         $name = $request->get_param('name');
+        $turnstile_token = $request->get_param('turnstileToken');
 
-        $user = Password_Auth::register($email, $password, $name);
+        $user = Password_Auth::register($email, $password, $name, $turnstile_token);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\ZenEyer\Auth\Auth\Password_Auth' in method 'register'. (undefined)

(StaticAccess)